File: 2021/test/day_25_test.rb
# frozen_string_literal: true

require "test_helper"
require "day_25"

class Day25Test < Minitest::Test
  def test_part1
    day = Day25.new

    assert_equal 58, day.part1(input("day_25_example_1"))
    assert_equal 453, day.part1(input("day_25"))
  end

  def test_part2
    day = Day25.new

    assert_equal 1, day.part2(input("day_25_example_1"))
    # assert_equal 1, day.part2(input("day_25"))
  end
end

File: 2022/test/day_25_test.exs
defmodule Day25Test do
  use ExUnit.Case

  import TestHelper
  import Day25

  test "part1" do
    assert part1(input("day_25_example_1")) == "2=-1=0"
    assert part1(input("day_25")) == "121=2=1==0=10=2-20=2"
  end

  test "part2" do
    # assert part2(input("day_25_example_1")) == 123
    # assert part2(input("day_25")) == 456
  end
end
